scripts/zones/Bastok_Markets_[S]/npcs/Kevan.lua

```diff
@@ -0,0 +1,46 @@
+-----------------------------------
+-- Area: Bastok Markets [S]
+--  NPC: Kevan
+-- !pos -308.590 -012.000 -094.227 189
+-- Sealed Container NPC
+-----------------------------------
+local ID = require("scripts/zones/Bastok_Markets_[S]/IDs")
+-----------------------------------
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
```

Comment:
Since the logic is all the same for these 3 NPCs, how much additional stuff is there for SCNMs to be added in the future? Does it warrant a global? or is it just dropped item -> key item -> go pop and fight monster?